### PR TITLE
add diligentgraphics-spirv-headers

### PR DESCRIPTION
**Pull request by AndreyMlashkin**
Specify library name and version:  **diligentgraphics-spirv-headers**

This lib is needed for Diligent-core recipe. Relates to https://github.com/conan-io/conan-center-index/pull/7804

---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by madebr — DISMISSED** [review]

File: recipes/diligentgraphics-spirv-headers/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
+from conans import ConanFile, tools, CMake
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class SpirvheadersConan(ConanFile):
+    name = "diligentgraphics-spirv-headers"
+    homepage = "https://github.com/DiligentGraphics/"
```
Comment (uilianries) [inline recipes/diligentgraphics-spirv-headers/all/conanfile.py:9]:
```suggestion
    homepage = "https://github.com/DiligentGraphics/SPIRV-Headers"
```

**Review by uilianries — DISMISSED** [review]

**Review by madebr — APPROVED** [review]

**Comment by AndreyMlashkin** [comment]
@uilianries @SSE4 , could you please approve? Other PRs are depending on this and pending.

**Review by SSE4 — APPROVED** [review]

**Review by uilianries — APPROVED** [review]
LGTM

**Comment by SpaceIm** [comment]
Why creating another recipe? This commit exists in the main Spirv-Cross repository: https://github.com/KhronosGroup/SPIRV-Headers/tree/5ea2d62e8c0ddd9e2a7d0ca5e3f2335e09e5f408

**Comment by AndreyMlashkin** [comment]
I will check on Monday. Thanks for checking. I will try to re-create all diligent recipes and try building diligent-core

**Pull request merged**